packages/kbn-optimizer/limits.yml

```diff
@@ -46,7 +46,7 @@ pageLoadAssetSize:
   lens: 96624
   licenseManagement: 41817
   licensing: 39008
-  lists: 202261
+  lists: 228500
```

Comment:
Can this added 26,239 be removed from the securitySolution plugin defined here?

I understand it's outside the scope of this PR, but I do want to advocate and ask that your team works to reduce these limits in the near future. At the very highest end, we believe these should be under 100k. The files reported here are served whenever Kibana is loaded, affecting the performance of Kibana as a whole. There are details about these metrics [here](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html) and details on how to reduce the sizes [here](https://www.elastic.co/guide/en/kibana/master/plugin-performance.html). Feel free to reach out if you have any questions.

Reply:
Looking at ways to decrease size. So the example given in the link deals with lazy loading components on register. If there isn't a dedicated exceptions parent page that loads these components, where is the best place to ask that they be lazily loaded?

The lists plugin currently does not register with core.

Reply:
Not sure what you mean by the lists plugin doesn't currently register with core. It's doing that [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/lists/public/plugin.ts#L17), which is why this bundle is being generated and included on every page refresh. How is this "lists" plugin currently being used?

Reply:
And we can take this part of the discussion to a dedicated issue. Bringing up the overall bundle sizes is to bring awareness to this increasing problem, which we across Kibana have been trying to reduce for a while now.

As far as this PR, if we have in-fact move logic from the `securitySolution` to the `lists` plugin, I would like to see that reflected in the limits file. If we are increasing `lists`, we should in theory be able to remove that added size from `securitySolution`.